docs/reference/mapping/types/nested.asciidoc

```diff
@@ -183,17 +183,26 @@ or <<request-body-search-stored-fields, `stored_fields`>>.
 
 The following parameters are accepted by `nested` fields:
 
-[horizontal]
 <<dynamic,`dynamic`>>::
-
-    Whether or not new `properties` should be added dynamically to an existing
-    nested object.  Accepts `true` (default), `false` and `strict`.
+(Optional, string)
+Whether or not new `properties` should be added dynamically to an existing
+nested object.  Accepts `true` (default), `false` and `strict`.
 
 <<properties,`properties`>>::
-
-    The fields within the nested object, which can be of any
-    <<mapping-types,datatype>>, including `nested`. New properties
-    may be added to an existing nested object.
+(Optional, object)
+The fields within the nested object, which can be of any
+<<mapping-types,datatype>>, including `nested`. New properties
+may be added to an existing nested object.
+
+`include_in_parent`::
+(Optional, boolean)
+If `true`, inner objects are indexed as <<object,flat key-value pairs>> for the
```

Comment:
To me, this description could be hard to understand. An alternate suggestion: "If `true`, all fields in the nested object are also added to the parent document as standard (flat) fields." A similar suggestion applies to `include_in_root` below.

Reply:
Thank you for the feedback @jtibshirani. I've reworded as suggested with 817c0ffcfca.